app/assets/javascripts/discourse/lib/to-markdown.js.es6

```diff
@@ -126,6 +126,53 @@ export class Tag {
     };
   }
 
+  static aside() {
+    return class extends Tag {
+      constructor() {
+        super("aside");
+        this.gap = "\n\n";
+      }
+
+      decorate(text) {
+        const parent = this.element.parent;
+
+        if (this.name === "p" && parent && parent.name === "li") {
+          // fix for google docs
+          this.gap = "";
+        }
+
+        return `${this.gap}${this.prefix}${text}${this.suffix}${this.gap}`;
+      }
+
+      toMarkdown() {
+        if (!this.element.attributes.class.split(" ").includes("quote")) {
+          return super.toMarkdown();
+        }
+
+        const blockquote = this.element.children.find(
```

Comment:
Is is possible that there are no `blockquote` as children of the `aside`?

Reply:
Yes, that is a possibility. Added a check.